### PR TITLE
Merge ChangeLog for r2020a

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Remove the ChangeLog entry for R2019b revision 2 and merge it with the one of R2020a.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:eyes:

File: docs/reference/changelog-r2020.md [hunk 1]
```diff
@@ -13,19 +17,34 @@ Released on XXX YYth, 2019.
...
-    - Split the Webots and controller libraries to avoid possible conflicts with external libraries.
     - Set the [ABB IRB 4600/40](../guide/irb4600-40.md) root node to [Robot](robot.md) instead of [Solid](solid.md) to be able to insert it everywhere.
+    - Webots now waits for extern controllers if the `Robot.synchronization` field is set to `TRUE`.
+    - Device names are displayed in the scene tree next to the node name.
+    - Split the Webots and controller libraries to avoid possible conflicts with external libraries.
     - Windows/Linux: Move the `Check for updates...` menu from `Tools` to `Help` for consistency with other applications.
-  - New Samples
-    - Added a `complete_apartment` world.
+  - Bug fixes
+    - Fixed [Lidar](lidar.md) point cloud access in controllers (thanks to Alexander).
```
Comment (DavidMansolino) [inline docs/reference/changelog-r2020.md]:
```suggestion
    - Fixed the [`wb_supervisor_node_reset_physics()`](supervisor.md#wb_supervisor_node_reset_physics) function when applied on dynamic objects (in motion and containing joints).
    - Fixed [Lidar](lidar.md) point cloud access in controllers (thanks to Alexander).
```

Comment (stefaniapedrazzi) [reply docs/reference/changelog-r2020.md]: Thank you. Probably the branch was merged after I created this change.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**